build/common/openwisp/settings.py

```diff
@@ -386,3 +386,5 @@
         INSTALLED_APPS.remove('openwisp_monitoring.device')
     if 'openwisp_monitoring.check' in INSTALLED_APPS:
         INSTALLED_APPS.remove('openwisp_monitoring.check')
+if EMAIL_BACKEND == 'djcelery_email.backends.CeleryEmailBackend':
+    INSTALLED_APPS.append('djcelery_email')
```

Comment:
Email is being sent, without adding any `pip install` statement for `djcelery_email`.

Reply:
let's add it anyway because we may remove djcelery_email from the requirements of openwisp-monitoring (since it's not directly used by openwisp-monitoring but it's a deployment detail, so it makes sense to define it here).

You can add it here: https://github.com/openwisp/docker-openwisp/blob/master/build/openwisp_base/requirements.txt, use a relaxed bound (not too relaxed).